assets/css/edd-admin.css

```diff
@@ -2344,6 +2344,11 @@ body.download_page_edd-reports {
 	right: 6px;
 }
 
+.edd-reports-wrapper .postbox h2 {
```

Comment:
Can we combine these two rules together? Sorry, should have asked for that before. The comment is helpful (there are likely more extensions than SL which use `h3` too), so I would keep that.